modules/programs/offlineimap.nix

```diff
@@ -90,6 +90,16 @@ in {
     programs.offlineimap = {
       enable = mkEnableOption "OfflineIMAP";
 
+      package = mkPackageOption pkgs "offlineimap" {
+        example = ''
+          pkgs.offlineimap.overridePythonAttrs (
+            old: {propagatedBuildInputs = old.propagatedBuildInputs ++
+             [pkgs.python3Packages.requests_oauthlib
+              pkgs.python3Packages.xdg
+              pkgs.python3Packages.gpgme];})'';
```

Comment:
Perhaps

```suggestion
          pkgs.offlineimap.overridePythonAttrs (old: {
            propagatedBuildInputs = old.propagatedBuildInputs
              ++ with pkgs.python3Packages; [ requests_oauthlib xdg gpgme ];
            })
```

Reply:
I had to add brackets around the whole statement to get it to work.